Externals.cfg

```diff
@@ -72,5 +72,12 @@ repo_url = https://svn-ccsm-models.cgd.ucar.edu/ww3/release_tags
 local_path = components/ww3
 required = False
 
+[norcpm]
+tag = n2.0.8
+protocol = git
+repo_url = https://github.com/NorESMhub/norcpm_esp
```

Comment:
Does this repository exist? Please create it (I can help if you do not have permissions).
Should it be created from https://github.com/BjerknesCPU/norcpm_esp ?

Reply:
Thanks, I created it.